### PR TITLE
Expand assignment graded result

### PR DESCRIPTION
**Pull request by garemoko**
Also adds underscore for the quiz attempt grade items

### CODE REVIEW

**Comment by garemoko** [comment]
@ryansmith94 this one is ready for review too

**Comment by garemoko** [comment]
https://github.com/jlowe64/moodle-logstore_xapi/issues/34

File: tests/AttemptReviewedTest.php [hunk 1]
```diff
@@ -39,8 +39,8 @@ protected function assertAttempt($input, $output) {
     }
 
     protected function assertGradeItems($input, $output) {
-        $this->assertEquals((float) $input['gradeitems']->grademin, $output['attempt_score_min']);
-        $this->assertEquals((float) $input['gradeitems']->grademax, $output['attempt_score_max']);
-        $this->assertEquals(($input['attempt']->sumgrades >= $input['gradeitems']->gradepass), $output['attempt_success']);
+        $this->assertEquals((float) $input['grade_items']->grademin, $output['attempt_score_min']);
+        $this->assertEquals((float) $input['grade_items']->grademax, $output['attempt_score_max']);
+        $this->assertEquals(($input['attempt']->sumgrades >= $input['grade_items']->gradepass), $output['attempt_success']);
```
Comment (ryansmith94) [inline tests/AttemptReviewedTest.php]: Is there potentially more to test in here? Like completed, scaled, etc.

Comment (garemoko) [inline tests/AttemptReviewedTest.php]: completion is addressed elsewhere. i can add scaled

**Pull request merged**